packages/fe-mockserver-core/src/data/entitySets/entitySet.ts

```diff
@@ -79,8 +79,11 @@ function transformationFn(type: string, check?: any) {
             return (data: string) => {
                 return data.endsWith(prepareLiteral(check, 'Edm.String') as string);
             };
-        case 'contains':
         case 'substringof':
+            return (data: string) => {
+                return check.indexOf(prepareLiteral(data, 'Edm.String') as string) !== -1;
```

Comment:
When is it `data` and when `check`?

Reply:
susbtringof(piece of string to check, string where to check) 

contains(string where to check, piece of string to check)

data equals the first arg
check equals the second arg

So for substringof check is the data model object and data is the static string to check (or whatever really)

Reply:
Okay. I didn't look at the `makeTransformationFn`. Now I get it (I hope 😄)